Gaijinizer patch/Map101.txt

```diff
@@ -254,9 +262,9 @@ I have prepared something like "Musk filter function".」#MTLed
 > CONTEXT: Map101/events/15/pages/0/28/Dialogue
 > CONTEXT: Map101/events/16/pages/0/28/Dialogue
 > CONTEXT: Map101/events/17/pages/0/28/Dialogue
-*「This is when such a scene is encountered
-\\C[2] Ability to display another picture and ease the
-expression.」#MTLed
+*「When you're confronted with one of those scenes, the
+   filter \\C[2]will cover the screen so you don't have to
+   actually see anything.」#Ron
```

Comment:
This isn't in the original text, but I think it's a nice way to clean up the visual text a bit...

Add a \\C[0] before the final angle bracket on this and all the other examples of red text coming up. This will make the final bracket white, which makes it look cleaner. There's a few times where this happens in the Bear's text.